editor/editor_node.cpp

```diff
@@ -317,36 +317,39 @@ void EditorNode::shortcut_input(const Ref<InputEvent> &p_event) {
 
 	Ref<InputEventKey> k = p_event;
 	if ((k.is_valid() && k->is_pressed() && !k->is_echo()) || Object::cast_to<InputEventShortcut>(*p_event)) {
-		EditorPlugin *old_editor = editor_plugin_screen;
-
 		if (ED_IS_SHORTCUT("editor/filter_files", p_event)) {
 			FileSystemDock::get_singleton()->focus_on_filter();
 			get_tree()->get_root()->set_input_as_handled();
```

Comment:
This should probably be handled with a `bool`, so a `bool is_handled = false;`, a `is_handled = true;` in each placez, and a check at the end.

Reply:
Done!